mycroft/skills/intent_services/adapt_service.py

```diff
@@ -0,0 +1,247 @@
+# Copyright 2020 Mycroft AI Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""An intent parsing service using the Adapt parser."""
+from adapt.context import ContextManagerFrame
+from adapt.engine import IntentDeterminationEngine
+from adapt.intent import IntentBuilder
+
+import time
+
+from mycroft.util.log import LOG
+from .base import IntentMatch
+
+
+class AdaptIntent(IntentBuilder):
+    def __init__(self, name=''):
+        super().__init__(name)
+
+
+def workaround_one_of_context(best_intent):
+    """ Handle Adapt issue with context injection combined with one_of.
```

Comment:
potentially not worth changing in this PR, but the latest version of adapt has addressed this issue!

https://github.com/MycroftAI/adapt/pull/103

Reply:
I almost forgot about that, I'll handle it in this PR as it's quite a simple change

Reply:
This has been added now